### DISCUSSION TITLE
Sort language codes in translations interface

### DISCUSSION THREAD

**Discussion by viters** [directus/directus#11219] [Feature Requests]
I'd suggest allowing to override sorting of languages in translations interface. It could work as with others sortable via panel collections, I add `sort` field and set it to be sorting field for collection.

app/src/interfaces/translations/translations.vue:252
It should either allow to set manual sorting or behave differently if `sort` exists on languages.

**Comment by azrikahar** [comment]
Just adding https://github.com/directus/directus/discussions/10478 here as a relevant discussion on this topic, with [PRs mentioned here](https://github.com/directus/directus/discussions/10478#discussioncomment-1802641) that has different approach but (presumably) targeting the same pain point.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
